ddmark/validation_integration_test.go

```diff
@@ -166,70 +166,148 @@ exclusivefieldstest:
 		})
 	})
 
-	Context("LinkedFields Marker", func() {
+	Context("LinkedFieldsValue Marker", func() {
 		It("checks out valid all-non-nil values", func() {
-			linkedfieldsYaml := `
-linkedfieldstest:
-  strfield: aa
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
+  strfield: aaa
   pstrfield: bb
   intfield: 1
   pintfield: 1
   aintfield: [1,2]
 `
-			err := validateString(linkedfieldsYaml)
+			err := validateString(linkedfieldsvalueYaml)
 			Expect(err.Errors).To(HaveLen(0))
 		})
+		It("rejects invalid requires value for StrField", func() {
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
+  strfield: notaaa
+  pstrfield: bb
+  intfield: 1
+  pintfield: 1
+  aintfield: [1,2]
+`
+			err := validateString(linkedfieldsvalueYaml)
+			Expect(err.Errors).To(HaveLen(1))
+		})
 		It("checks out valid all-nil values", func() {
-			linkedfieldsYaml := `
-linkedfieldstest:
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
   randomintfield: 1
   strfield:
   pstrfield:
   intfield: 0 # is nil
   pintfield:  # is nil
   aintfield:
 `
-			err := validateString(linkedfieldsYaml)
+			err := validateString(linkedfieldsvalueYaml)
 			Expect(err.Errors).To(HaveLen(0))
 		})
 		It("rejects both errors - first fields", func() {
-			linkedfieldsYaml := `
-linkedfieldstest:
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
   strfield: aa
   pstrfield: aa
-  intfield:
+  intfield: 1
   pintfield:
   aintfield:
 `
-			err := validateString(linkedfieldsYaml)
+			err := validateString(linkedfieldsvalueYaml)
 			Expect(err.Errors).To(HaveLen(2))
 		})
 		It("rejects both errors - second fields", func() {
-			linkedfieldsYaml := `
-linkedfieldstest:
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
   strfield:
   pstrfield:
   intfield: 1  # is non-nil
   pintfield: 0 # is non-nil
   aintfield:
 `
-			err := validateString(linkedfieldsYaml)
+			err := validateString(linkedfieldsvalueYaml)
 			Expect(err.Errors).To(HaveLen(2))
 		})
 		It("rejects one error - 0 value is nil on pointer", func() {
-			linkedfieldsYaml := `
-linkedfieldstest:
-  strfield: aa
+			linkedfieldsvalueYaml := `
+linkedfieldsvaluetest:
+  strfield: aaa
   pstrfield: aa
   intfield: 0 	# is nil
   pintfield: 0  # is non-nil
   aintfield: [1,2]
 `
-			err := validateString(linkedfieldsYaml)
+			err := validateString(linkedfieldsvalueYaml)
 			Expect(err.Errors).To(HaveLen(1))
 		})
 	})
 
+	Context("LinkedFieldsValueWithTrigger Marker", func() {
+		// +ddmark:validation:LinkedFieldsValueWithTrigger={StrField=aaa,IntField=2}
+		// +ddmark:validation:LinkedFieldsValueWithTrigger={PStrField=bbb,PIntField=12,AIntField}
+		It("checks out valid all-incorrect initial values", func() {
+			linkedfieldsvaluewithtriggerYaml := `
+linkedfieldsvaluewithtriggertest:
+  strfield: aa      # invalid
+  pstrfield: bb     # invalid
+  intfield: 12
+  pintfield: 1
+  aintfield: [1,2]
+`
+			err := validateString(linkedfieldsvaluewithtriggerYaml)
+			Expect(err.Errors).To(HaveLen(0))
```

Comment:
```suggestion
			Expect(err.Errors).To(BeEmpty())
```

FYI this can be used also